spec/shr_datatype_vs.txt

```diff
@@ -29,36 +29,19 @@ ValueSet: 				CodingQualifierVS
 #default_code			"The code was supplied by the system by default, because no specific coded value was available."
 */
 
-
-ValueSet:           BodyPositionVS
-Description:        "Human body positions associated with actions or observations."
-SCT#40199007	    "Supine body position"
-SCT#33586001	    "Sitting position"	
-SCT#10904000	    "Orthostatic body position"
-SCT#414585002	    "Left lateral tilt"	
-SCT#415346000	    "Right lateral tilt"
-SCT#30212006	    "Fowler's position"	
-SCT#26527006	    "Inverse Trendelenburg position"
-SCT#102536004	    "Left lateral decubitus position"	
-SCT#1240000	        "Prone body position"	
-SCT#102538003	    "Recumbent body position"
-SCT#423413008	    "Reverse trendelenburg positioning"	
-SCT#102535000	    "Right lateral decubitus position"
-SCT#34106002	    "Trendelenburg position"
-
 ValueSet:				ClockDirectionVS
 Description:		"The relative direction of an object described using the analogy of a 12-hour clock to describe angles and directions. One imagines a clock face lying either upright or flat in front of oneself, and identifies the twelve hour markings with the directions in which they point."
 Includes codes descending from SCT#272485000 "Clockface position (qualifier value)"
 
 ValueSet:		UpperBoundTypeVS
 Description:	"Indicates the actual value is lt (<) or lte (<=) the given value."
-UGLY#<			"The actual value is less than the given value."
-UGLY#<=			"The actual value is less than or equal to the given value."
+COMP#<			"The actual value is less than the given value."
```

Comment:
interesting.  creating a value set of comparative symbols. The current model doesn't have an existing reference to it so I'd be curious to see how it'll work in an implementation if the need arises.